internal/server/volume/server.go

```diff
@@ -69,37 +49,41 @@ func (s *Server) MountVolume(context context.Context, request *internal.MountVol
 		klog.Errorf("volume id empty")
 		return response, fmt.Errorf("volume id empty")
 	}
-	path := request.Path
-	if path == "" {
-		klog.Errorf("mount id empty")
-		return response, fmt.Errorf("mount path empty")
+	targetPath := request.TargetPath
+	if targetPath == "" {
+		klog.Errorf("targetPath empty")
+		return response, fmt.Errorf("MountVolumeRequest.TargetPath empty")
 	}
 
-	err := s.hostAPI.MountVolume(volumeID, path)
+	err := s.hostAPI.MountVolume(volumeID, targetPath)
 	if err != nil {
 		klog.Errorf("failed MountVolume %v", err)
 		return response, err
 	}
 	return response, nil
 }
 
-func (s *Server) DismountVolume(context context.Context, request *internal.DismountVolumeRequest, version apiversion.Version) (*internal.DismountVolumeResponse, error) {
-	klog.V(5).Infof("DismountVolume: Request: %+v", request)
-	response := &internal.DismountVolumeResponse{}
+func (s *Server) DismountVolume(context.Context, *internal.DismountVolumeRequest, apiversion.Version) (*internal.DismountVolumeResponse, error) {
+	return nil, fmt.Errorf("DismountVolume is deprecated, use UnmountVolume")
```

Comment:
This is the first time we are making the Deprecation announcement. We should not block an existing API from working. Would a DismountVolume from a previous API version get routed to UnmountVolume?

Reply:
got it, yes, I'm routing it to `UnmountVolume` instead